plugins/catalog-backend/src/next/Stitcher.ts

```diff
@@ -137,7 +142,15 @@ export class Stitcher {
             ['backstage.io/orphan']: 'true',
           };
         }
-
+        if (errors !== '') {
+          const parsedErrors = JSON.parse(errors);
```

Comment:
Let's try catch this to avoid affecting stitching if we can't parse errors.

Reply:
this really should not happen :P